docs/paas-plus/dev/integrate-spryker-code-upgrader.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Integrate Spryker Code Upgrader into your development process
+description: How to integrate Spryker Code Upgrader into your development process
+template: concept-topic-template
+---
+
+When integrating Spryker Code Upgrader into your development process it is important to consider your development process and integrate it correctly, here we would like to briefly explain how Spryker Code Upgrader can be integrated into your Scrum development process and Git flow.
+
+![Spryker CI projects](https://spryker.s3.eu-central-1.amazonaws.com/docs/paas%2B/dev/integrate-spryker-code-upgrader.md/spryker-code-upgrader-integration-diagramm.png)
+
+## Scrum
+
+* Sprint Planning: The team can review the upgrade suggestions as pull requests in your repository hosting service and prioritize the software upgrades that need to be installed in the upcoming sprint.
+
+* Sprint Execution: During the sprint, the team can review, validate, modify and merge the upgrades using the new process offered by Spryker Code Upgrader.
```

Comment:
What do you mean by the "new process"?

Reply:
Probably the process that is displayed on the previous diagram - https://spryker.atlassian.net/wiki/spaces/DX/pages/3671916918/Integrate+Spryker+Code+Upgrader+into+your+development+process

The SCRUM process when every sprint development team plans also the portion of the updates provided by Spryker Code Upgrader